hedyweb.py

```diff
@@ -28,7 +28,10 @@ def get_translations(self, language):
 class PageTranslations:
   def __init__(self, page):
     self.data = {}
-    translations = glob.glob('content/pages/' + page + '/*.yaml')
+    if page in 'start, learn-more, for-teachers':
```

Comment:
Shouldn't this be a list like this: `['start', 'learn-more', 'for-teachers']`?

Reply:
Everythings seems to work in Python :-)

Reply:
That's because it's a string comparison in this case which is dangerous. Because 'a' in ... is also valid as the character can be found in the string. With a in list comparison we only validate to True if it matches one of the items exactly.